Baby Monitor/Source Files/Modules/Onboarding/Pairing/OnboardingPairingCoordinator.swift

```diff
@@ -136,4 +148,15 @@ private extension OnboardingPairingCoordinator {
         .disposed(by: viewModel.bag)
         navigationController.pushViewController(viewController, animated: true)
     }
+
+    func showCompareCodeView(with url: URL) {
+        let viewModel = OnboardingCompareCodeViewModel(
+            webSocketEventMessageService: appDependencies.webSocketEventMessageService,
+            urlConfiguration: appDependencies.urlConfiguration,
+            serverURL: url,
+            activityLogEventsRepository: appDependencies.databaseRepository)
+        let viewController = OnboardingCompareCodeViewController(viewModel: viewModel
+        )
```

Comment:
This could be in the same line as the rest

Reply:
😅